### DISCUSSION TITLE
If cloning is possible, documentation can be updated.

### DISCUSSION THREAD

**Discussion by Verhoeckx** [cockpit-project/cockpit-machines#271] [General]
Not very important but if cloning is possible, the table on this web page can be updated:

Web page
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/managing-virtual-machines-in-the-web-console_system-management-using-the-rhel-8-web-console#differences-between-virtualization-features-in-virtual-machine-manager-and-the-rhel-8-web-console_managing-virtual-machines-in-the-web-console

Table
Table 27.2. VM managemennt tasks that cannot be performed using the web console in RHEL 8.0.
Cloning a virtual machine | CURRENTLY UNAVAILABLE | virt-clone

**Comment by marusak** [comment]
This needs to be reported to documentation team. Happy to do that in your stead.
@KKoukiou can you please check if that table is up to date or if there are more not updated items?

**Reply by KKoukiou** [reply]
The documentation is correct. Table 27.2. VM managemennt tasks that cannot be performed using the web console in RHEL 8.0

in RHEL 8.0 this was not supported indeed, was intruduced in https://cockpit-project.org/blog/cockpit-234.html which is available in a later RHEL.

**Reply by Verhoeckx** [reply]
But in the second column you could add that is it was added to RHEL 8.4?
RHEL 8.4 comes with version 238.2 of Cockpit (`dnf info cockpit`).

I don't mean literally you but someone who is responsible for the documentation.

**Reply by KKoukiou** [reply]
Check thanks for noting that, I guess you 're right. I will propagate it!

**Comment by ParthS18** [comment]
Hello @Verhoeckx, @marusak, and @KKoukiou,

Thanks a lot for the heads-up! I've updated [the documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/managing-virtual-machines-in-the-web-console_system-management-using-the-rhel-8-web-console#differences-between-virtualization-features-in-virtual-machine-manager-and-the-rhel-8-web-console_managing-virtual-machines-in-the-web-console).

Cheers,
Parth.
